e2e/js_image/BUILD.bazel

```diff
@@ -1,19 +1,37 @@
 load("@npm//:defs.bzl", "npm_link_all_packages")
-load("@aspect_bazel_lib//lib:docs.bzl", "stardoc_with_diff_test", "update_docs")
-load("@aspect_rules_js//js:defs.bzl", "js_binary")
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@aspect_rules_js//js:defs.bzl", "js_binary", "js_image_layer")
+load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")
 load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 load("@io_bazel_rules_docker//contrib:test.bzl", "container_test")
 load("@io_bazel_rules_docker//container:layer.bzl", "container_layer")
```

Comment:
hm, at what point would we use rules_oci here? Maybe it will be another package under this same e2e that shows the usage with those rules? I suppose ppl will want to see usage with rules_docker for at least a year while the migrations are happening

Reply:
there's already a js example in rules_oci. https://github.com/bazel-contrib/rules_oci/tree/main/example/js. perhaps i should move it here?

Reply:
The example is also here to provide test coverage right? If rules_js changes could break it, then having an example here is good. If rules_oci changes might break it, then we might *also* want to have an example over there?

Reply:
Yes, examples here are for rules_js test coverage. It's less likely that rules_oci will break just rules_js given it doesn't care about what creates the image layers.  Nevertheless, I think we should have both.

I'll add a rules_oci example here as well.